langchain/document_loaders/utils/metadata.py

```diff
@@ -0,0 +1,83 @@
+"""Interface for retrieving metadata from the various document loaders."""
```

Comment:
Anything related to metadata retrieval and can placed here. And is a good place to help standardize across the various document loaders.

One thing I have to say about this script is that if we were to include support for all the document loaders, e.g.  twitter, reddit, pdf, url, unstructured, etc., this will be a very very long script. Not too sure what are the appropriate design guidelines here.

Or would it be better to just have the metadata retrieval inside the individual document loader scripts?